homeassistant/components/renson_endura_delta/__init__.py

```diff
@@ -0,0 +1,117 @@
+"""The Renson Endura Delta integration."""
+from __future__ import annotations
+
+import rensonVentilationLib.generalEnum as rensonEnums
+import rensonVentilationLib.renson as renson
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, ServiceCall
+
+from .const import DOMAIN
+
+PLATFORMS = ["sensor"]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Renson Endura Delta from a config entry."""
+    hass.data.setdefault(DOMAIN, {})
+
+    hass.data[DOMAIN][entry.entry_id] = renson.RensonVentilation(entry.data["host"])
+
+    hass.config_entries.async_setup_platforms(entry, PLATFORMS)
+
+    return True
+
+
+async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Unload a config entry."""
+    unload_ok = await hass.config_entries.async_unload_platforms(entry, PLATFORMS)
+    if unload_ok:
+        hass.data[DOMAIN].pop(entry.entry_id)
+
+    return unload_ok
+
+
+def setup(hass: HomeAssistant, config: ConfigEntry) -> bool:
+    """Handle all the services of the Renson API."""
+
+    def handle_manual_level_set(call: ServiceCall):
+        service: renson.RensonVentilation = renson.RensonVentilation(
```

Comment:
This is not how it works. You should register entity services. However, the first PR should be limited to just config flow and a single platform, so let's leave these out.